src/NzbDrone.Core.Test/ParserTests/HashedReleaseFixture.cs

```diff
@@ -87,9 +87,22 @@ public class HashedReleaseFixture : CoreTest
                 "Movie Title",
                 Quality.HDTV720p,
                 null
+            },
+            new object[]
+            {
+                @"C:\Test\Movie.Title.2018.720p.HDTV.H.264\cwnOJYks5E2WP7zGuzPkdkRK3JkWw0.mkv".AsOsAgnostic(),
+                "Movie Title",
+                Quality.HDTV720p,
+                null
+            },
+            new object[]
+            {
+                @"C:\Test\Movie Title 1964 Criterion Collection 1080p Blu-ray Remux AVC DTS-HD MS 1 0\zM0vnVU1SoV4uwTihfzTHkYmJPBemgN3MqXG1fY.mkv".AsOsAgnostic(),
+                "Movie Title",
+                Quality.Remux1080p,
+                null
```

Comment:
These two cases pass just fine on develop currently without this change, given we don't get hung up on `s5E2` in the hash. So what does this actually do for us?

Reply:
really just following precedent of the previous obfuscation tests and logic in there

Reply:
Yea my point is why do we need this new hashed regex if its not doing anything for us.. at least based on the test cases added.